src/hooks/query/loginQuery.js

```diff
@@ -0,0 +1,8 @@
+import { useQuery } from 'react-query';
+import { fetchUsers } from '../../services/userService';
+
+export const useQueryToLogin = (config) => {
+    const response = useQuery('login', () => fetchUsers(), config);
```

Comment:
'login' should be a CONSTANT!!!!!!!!!!
config you can extend from your component, but let it be with some default value, for example {
            cacheTime: 0,
            refetchOnWindowFocus: false,
            ...config,
        }